pkg/plugins/builtin/http/http.go

```diff
@@ -31,6 +32,7 @@ type HTTPConfig struct {
 	HTTPBasicAuth  BasicAuth   `json:"basic_auth,omitempty"`
 	DenyRedirects  string      `json:"deny_redirects,omitempty"`
 	Parameters     []Parameter `json:"parameters,omitempty"`
+	MagicPrefix    string      `json:"magic_prefix,omitempty"`
```

Comment:
TrimPrefix instead of "Magic" ? More specific regarding what is expected.

Reply:
done